src/towncrier/test/helpers.py

```diff
@@ -0,0 +1,50 @@
+from functools import wraps
+from pathlib import Path
+
+from click.testing import CliRunner
+
+
+def read(filename):
```

Comment:
with type annotations this probably doesn't even need a docstring

Reply:
Yeah I was torn. Adding a docstring would make it more difficult to understand so I've decided against the hobgoblin. :)

Reply:
I have added type hints to helpers, to "document" them. I've YOLO'ed the decorator typing to avoid depending on typing_extentions for now.